src/core/mac/sub_mac.hpp

```diff
@@ -623,7 +609,7 @@ class SubMac : public InstanceLocator, private NonCopyable
 #if OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE
     TimerMicro mTimer;
 #else
-    TimerMilli mTimer;
+    TimerMilli                mTimer;
```

Comment:
`clang-format` is changing this. I have no idea why. I guess it is trying to align it with something?